contracts/GuestList.vy

```diff
@@ -69,10 +70,7 @@ bancor: Bancor
 @external
 def __init__():
     self.bouncer = msg.sender
-    self.activation = block.timestamp
-    # constants
-    self.min_bag = 10 ** 18  # 1 YFI to enter
-    self.ape_out = 30 * 86400  # 30 days falloff
+    self.bribe_cost = 25 * 10 ** 15  # 0.025 YFI life pass
```

Comment:
you can also do `25 * 1e15`

Reply:
Vyper says "Cannot perform multiplication between dislike types"